src/run.cr

```diff
@@ -23,7 +23,14 @@ module Run
     JSON.parse(expect_json_file)
   end
 
-  def self.run_test(exercise_index : Int, file_name : String, run_with_stdin = true)
+ def self.get_command(initial_command : JSON::Any, file_name : String) : String
+    initial_command = initial_command.to_s
+
+    return initial_command.sub("$", file_name) if initial_command.includes?("$")
+    return "#{initial_command} #{file_name}"
+  end
+
+  def self.run_test(exercise_index : Int, file_name : String, run_with_stdin = true, run_with_verbose = true)
```

Comment:
I don’t think verbose should be enabled by default

Reply:
Thanks for the suggestion, I changed the default value